relm4-macros/tests/simple.rs

```diff
@@ -69,10 +68,3 @@ impl SimpleComponent for AppModel {
         }
     }
 }
-
-fn assert_impls_debug<T: std::fmt::Debug>() {}
-
-#[test]
```

Comment:
Why remove this test?

Reply:
The debug impl assert should only be part of one test, I think. It doesn't really make sense to check this several times in different test cases. So I've added this to the simple test again and removed it from the other two test cases which used it (due to copy-pasting the simple example).

Reply:
Sounds good.